src/Components/Form.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { ImInfo } from "react-icons/im";
+function Form() {
+  const [breed, setBreed] = useState("");
+  const [weight, setWeight] = useState("");
+  const [height, setHeight] = useState("");
+  const [isInfo, setIsInfo] = useState(false);
+  const [heightUnits, setHeightUnits] = useState("in");
+  const [weightUnits, setWeightUnits] = useState("lb");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(breed, weight, height, heightUnits, weightUnits);
+  };
+
+  const toggleWeight = (value) => {
+    setWeightUnits(value);
+  };
+
+  const toggleHeight = (value) => {
+    setHeightUnits(value);
+  };
+  return (
+    <FormDiv onSubmit={handleSubmit}>
+      <label>What is your dog's breed?</label>
+      <input onChange={(e) => setBreed(e.target.value)}></input>
+      <div className="container">
+        <label>What is your dog's weight?</label>
+        <button
+          className={weightUnits === "lb" ? "active" : ""}
+          onClick={() => toggleWeight("lb")}
+        >
+          lb
+        </button>
+        <button
+          className={weightUnits === "kg" ? "active" : ""}
+          onClick={() => toggleWeight("kg")}
+        >
+          kg
+        </button>
+      </div>
+      <input
+        className="shorten-input"
+        onChange={(e) => setWeight(e.target.value)}
+      ></input>
+      <div className="container">
+        <label>What is your dog's height?</label>
+        <StyledInfoButton onClick={() => setIsInfo(!isInfo)} />
+        <button
+          className={heightUnits === "in" ? "active" : ""}
+          onClick={() => toggleHeight("in")}
```

Comment:
okay, I think I get why this is called toggle now. I would still think about adjusting the naming so it's a little more clear what the function is for and what it's doing though.

Reply:
ill change it to updateHeight and updateWeight